### DISCUSSION TITLE
Screen wake lock

### DISCUSSION THREAD

**Discussion by yurgh** [blakeblackshear/frigate#4237] [Ideas]
I sometimes view the cameras in the browser instead of a dedicated viewer on an android tablet or phone. It would be great if we could add a screen wake-lock to prevent the screen to go dim and turn off: https://web.dev/wake-lock/ Perhaps also a setting in options for enabling/disabling wake lock?

**Comment by Xedecimal** [comment]
One solution is to use Power Toys - Awake tool, then you can have a shortcut to turn it back off when you're going to bed or leaving when there'll be nobody looking at it just wasting power.

(also built into the web interface, will no longer keep things awake when the browser isn't activated or minimized)

**Comment by yurgh** [comment]
It is no longer an issue, as the GUI is completely rewritten now - the OP was three years ago :)
